db/deploy/views/summary_view.sql

```diff
@@ -138,6 +138,7 @@ FROM ccbc_public.application a
     comment on column ccbc_public.summary_view.archived_at is 'archived at timestamp';
     comment on column ccbc_public.summary_view.intake_id is 'Application Intake Number, used as the prefix for CCBC reference number';
 
+    grant execute on function ccbc_public.application_status to ccbc_auth_user;
```

Comment:
```suggestion
    grant execute on function ccbc_public.application_status to public;
```
Should execute be granted to `public`?  Metabase doesn't use `ccbc_auth_user`

Additionally, can you add a pgTap test, using https://pgtap.org/documentation.html#function_privs_are?